go-controller/pkg/ovn/service.go

```diff
@@ -36,11 +38,20 @@ func (ovn *Controller) deleteService(service *kapi.Service) {
 			targetPort = svcPort.Port
 		}
 
-		err := ovn.createLoadBalancerVIP(ovn.getLoadBalancer(protocol),
-			service.Spec.ClusterIP, svcPort.Port, ips, targetPort)
-		if err != nil {
-			logrus.Errorf("Error in deleting load balancer for service "+
-				"%s:%d %+v", service.Name, svcPort.Port, err)
+		if service.Spec.Type == "NodePort" {
```

Comment:
Same comment here about making a "nodeport" create a east-west load-balancing entry too in addition to gateway entry.